setuptools/independent/DepsProcessor/RuntimeReference.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.ProjectModel.Graph;
+using Microsoft.Extensions.DependencyModel;
+using NuGet.Frameworks;
+
+namespace DepsProcessor
+{
+    public class RuntimeReference
+    {
+        public static List<RuntimeLibrary> RemoveReferences(IReadOnlyList<RuntimeLibrary> runtimeLibraries, IReadOnlyList<string> refname)
```

Comment:
`refname` is a bad name for this parameter.  It should be plural, since it is a list.  And it should have full names, no abbreviations.

Since this is references we are talking about, how about just `names`?